modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -455,9 +453,9 @@ namespace {
             "Translation = {"
                 "Type = 'StaticTranslation',"
                 "Position = {"
-                        + std::to_string(galacticTarget.x) + ", "
-                        + std::to_string(galacticTarget.y) + ", "
-                        + std::to_string(galacticTarget.z) + ", "
+                    + std::to_string(galacticTarget.x) + ", "
```

Comment:
The `+` would move to the end of previous line instead to make visually sure that it is a continuation:

```cpp
"Position = {" +
    std::to_string(galacticTarget.x) + ", " +
    std::to_string(galacticTarget.y) + ", " +
```
```